src/shared/components/studyTagsTooltip/StudyTagsTooltip.tsx

```diff
@@ -43,12 +44,16 @@ export type StudyTagsTooltipProps = {
 export type BuildOverlayTooltipProps = {
     studyDescription: string;
     studyId: string;
+    isVirtualStudy: boolean;
 };
 
 @observer
 default class BuildOverlay extends React.Component<BuildOverlayTooltipProps, {}> {
     @observable readonly studyMetadata = remoteData({
         invoke: async () => {
+            if (this.props.isVirtualStudy) {
```

Comment:
you should just deal with this condition down in the render method instead of muddying the promise with it.